packages/universal/debug/src/setup.ts

```diff
@@ -2,18 +2,13 @@ import { defineDebug } from "@starbeam/reactive";
 
 import debug from "./debug.js";
 
-if (
-  (globalThis.Buffer as BufferConstructor | undefined) === undefined &&
-  typeof require === "function"
-) {
-  try {
-    // this is for CJS only, so require is the only option here
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const buffer = require("buffer") as { Buffer: BufferConstructor };
-    globalThis.Buffer = buffer.Buffer;
-  } catch {
-    // ignore
+if (import.meta.env.DEV) {
+  if (!import.meta.env.CJS) {
+    if ((globalThis.Buffer as BufferConstructor | undefined) === undefined) {
+        const buffer = await import("buffer") as { Buffer: BufferConstructor };
+        globalThis.Buffer = buffer.Buffer;
+    }
   }
-}
 
-defineDebug(debug);
+  defineDebug(debug);
```

Comment:
in both CJS and ESM, but only in DEV, we still want to `defineDebug` to get our nice stack traces